resource-managers/kubernetes/docker-minimal-bundle/src/main/docker/driver/Dockerfile

```diff
@@ -15,26 +15,13 @@
 # limitations under the License.
 #
 
-FROM openjdk:8-alpine
+FROM spark-base
 
 # If this docker file is being used in the context of building your images from a Spark distribution, the docker build
 # command should be invoked from the top level directory of the Spark distribution. E.g.:
 # docker build -t spark-driver:latest -f dockerfiles/driver/Dockerfile .
 
-RUN apk upgrade --update
-RUN apk add --update bash tini
-RUN mkdir -p /opt/spark
-RUN touch /opt/spark/RELEASE
-
-ADD jars /opt/spark/jars
-ADD examples /opt/spark/examples
-ADD bin /opt/spark/bin
-ADD sbin /opt/spark/sbin
-ADD conf /opt/spark/conf
-
-ENV SPARK_HOME /opt/spark
-
-WORKDIR /opt/spark
+COPY examples /opt/spark/examples
```

Comment:
The support for unpacking tarballs is actually one of the side-effects of ADD that is considered harder to reason about.  I think the idea is, just looking at a docker-file you may not be able to tell exactly _what_ operations that ADD is doing.  I don't have a real strong opinion on that.  It's arguably more convenient too.   But that's the current "official" position.